imapautofiler/actions.py

```diff
@@ -254,6 +254,44 @@ def invoke(self, conn, mailbox_name, message_id, message):
         )
 
 
+class Mark_as(Action):
+    """
+    Add flag to message
+    """
+
+    NAME = 'mark-as'
+    _log = logging.getLogger(NAME)
+
+    def __init__(self, action_data, cfg, flag):
+        super().__init__(action_data, cfg)
+        self.flag = flag
+
+    def report(self, conn, mailbox_name, message_id, message):
+        self._log.info()
+
+    def invoke(self, conn, mailbox_name, message_id, message):
+        conn.store(message_id, '+FLAG', '\\{}'.format(self.flag))
```

Comment:
Can self.flag be any string? Or does it need to be something specific to correspond to the IMAP standard? If it has to be one of a set of fixed values, I don't know if I'm comfortable relying on the user to pass it as an argument to the action correctly.

Reply:
I'll modify the class names in a minute. We'll raise an error if the incorrect flag is entered?

Reply:
What sorts of values are allowed for flag?

Reply:
seen, answered, flagged, deleted, draft, recent